daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -359,6 +359,43 @@ function Set-Repos {
   Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' addrepo 'google-compute-engine-stable' 'https://packages.cloud.google.com/yuck/repos/google-compute-engine-stable'
 }
 
+function Export-ImageMetadata {
+  $version = (Get-ComputerInfo).OsVersion
+  $family = 'windows-' + (Get-ComputerInfo).windowsversion
+  $name =  (Get-ComputerInfo).OSName
+  $release_date = (Get-Date).ToUniversalTime()
+  $image_metadata = @{'family' = $family;
+                      'version' = $edition;
+                      'name' = $name;
+                      'location' = ${script:outs_dir};
+                      'build_date' = $release_date;
+                      'build_repo' = $stage;
```

Comment:
I don't see the variable set anywhere

Reply:
This should be stable, as it has hardcode in workflow.